java/src/hdf/hdf5lib/H5.java

```diff
@@ -15497,6 +15497,24 @@ public synchronized static native String H5VLget_connector_name(long object_id)
     public synchronized static native void H5VLunregister_connector(long connector_id)
         throws HDF5LibraryException;
 
+    /**
+     * @ingroup JH5VL
+     *
+     * H5VLcmp_connector_cls Determines whether two connector identifiers refer to the same connector.
```

Comment:
This routine is necessary because it's possible to have VOL connector IDs that are different, but point to the same underlying VOL connector class.